doc/Makefile

```diff
@@ -1,10 +1,22 @@
-#DEPS=pervasives.satyh gr.satyh list.satyh tabular.satyh math.satyh color.satyh deco.satyh decoset.satyh vdecoset.satyh stdja.satyh local.satyh
+#!/usr/bin/make -f
 
-all:
-	satysfi doc-primitives.saty -o primitives.pdf
+SATYSFI = satysfi
 
-lang:
-	satysfi doc-lang.saty -o lang.pdf
+all: primitives.pdf lang.pdf math1.pdf
```

Comment:
I'd like to share with you a piece of code from my WIP branch for the same goal:
```make
.PHONY: all
all: primitives.pdf lang.pdf math1.pdf

.SUFFIXES: .saty .pdf
.saty.pdf:
        satysfi $< -o $@
```

Reply:
FYI, suffix rules is obsolete now (v. https://www.gnu.org/software/make/manual/html_node/Suffix-Rules.html) in favor of pattern rules like this.

```make
.PHONY: all
all: primitives.pdf lang.pdf math1.pdf

%.pdf: doc-%.saty
        satysfi $< -o $@
```

Reply:
Ah, ok, I'm fine if it is also supported by non-GNU variants too. math1.saty is not doc-prefixed, so we will need renaming anyway.

Reply:
Oh, I didn't realized POSIX standard does not contain pattern rules, therefore NetBSD make, OpenBSD make, and so on don't support it. (FreeBSD make does and Mac OS X uses GNU make, though)

It makes sense to stick at suffix rules in terms of portability.